news/issues/issue2.md

```diff
@@ -0,0 +1,52 @@
+
+
+### [Enable Spinnaker Usage Statistics Today!](https://www.spinnaker.io/community/stats/)
+
+In 2020, the vision of data-driven product development for Spinnaker has taken a big step forward with the implementation of Telemetry. For release 1.18.x, telemetry is opt-in, for testing and fine-tuning. You can help provide a clear picture by turning data collection on in your Spinnaker instance(s) now. As of 1.19.0+, telemetry will be enabled by default. See the linked community stats page for instructions and more info.
+
+
+### [Plugin Creators Guide](https://www.spinnaker.io/guides/user/plugin-users/)
+
+Armory and Netflix engineers have built a plug-in framework that enables extension of Spinnaker without the need to add custom stages. The new plugin system allows us to integrate services with Spinnaker outside of runtime, with clearly defined extension points to work from. The new Plugin Creators Guide walks us through creating and loading a simple plugin. Use it as a starting point to build your own!
+
+
+### [Continuous Delivery Summit Europe](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/colocated-events/#continuous-delivery-summit)
+
+The Continuous Delivery Summit will take place on March 30, 2020 in Amsterdam, as a Day 0 event at KubeCon EU. Hosted by the Continuous Delivery Foundation, this event will feature several talks from Spinnaker end-users and community members, as well as a project interoperability update and much more. Click the link for registration details.
+
+
+### [Future of SRE: Robert Keng Builds a DeploymentBot #withSpinnaker](https://blog.spinnaker.io/future-of-sre-robert-keng-builds-a-deploybot-withspinnaker-70ff3e37c56a)
+
+Hear from an interview with Robert Keng about a chatbot he built for Chime that deploys applications with Spinnaker. Learn how he managed constraints of developing against Slack, and his views on site reliability engineering, cloud native complexity, and the myth of DevOps.
+
+
+### [Continuous Delivery using Spinnaker on Amazon EKS](https://aws.amazon.com/blogs/opensource/continuous-delivery-spinnaker-amazon-eks/)
+
+In this AWS Open Source Blog post, Irshad Buchh outlines detailed instructions for installing and configuring Spinnaker via Halyard on Amazon EKS, proposing architecture for both CI and CD. He explains how to create an AWS Cloud9 environment and EKS clusters to host Spinnaker.
+
+### [Install & Manage Spinnaker on GCP](https://cloud.google.com/docs/ci-cd/spinnaker/spinnaker-for-gcp)
+
+Google’s guide to installing OSS Spinnaker on GCP explains how it adds value with a Cloud Shell-integrated management console, sample applications, and best practices for installing and running Spinnaker on GCP. This guided method offers an easy path to a production-ready configuration.
+
+
+### [First Look: Community Gardening Days April 9-10](https://www.armory.io/blog/first-spinnaker-community-gardening-days-april-9-10-salesforce/)
```

Comment:
Note that this link is broken as of now; it is scheduled to be published tomorrow, 2/18, at 7:45 AM PST

Reply:
And indeed, the link now works!